services/queue/schemas/v1/cancel-task-group-response.yml

```diff
@@ -0,0 +1,40 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Cancel Task-Group Response"
+description: |
+  Response from a `cancelTaskGroup` request.
+type:               object
+properties:
+  taskGroupId:
+    title:          "Task-Group Identifier"
+    description: |
+      Identifier for the task-group being listed.
+    type:           string
+    pattern:        {$const: slugid-pattern}
+  totalCount:
```

Comment:
I'm not sure if this is worth for inclusion as this number might change between calls, if new tasks are being added. But at the same time it could be an indicator that subsequent calls do something, as the number of tasks in the group grows

Reply:
Maybe `taskGroupSize`? Naming is so hard. I think it isn't obvious what "total" refers to...

Reply:
suggestion accepted! Thanks!